config.toml

```
@@ -68,20 +81,25 @@ sphinxopts = [
 
 [languages.pl]
 name = "Polish"
+translated_name = "Polski"
 
 [languages.pt_BR]
 name = "Brazilian Portuguese"
+translated_name = "Português Brasileiro"
```

It's not a proper name in pt_BR.

```suggestion
translated_name = "Português brasileiro"
```